server/src/main/scala/org/http4s/server/package.scala

```diff
@@ -82,8 +87,7 @@ package object server {
      * with status [[Status.NotFound]] for any requests where the function
      * is undefined.
      */
-    def apply(pf: PartialFunction[Request, Task[Response]]): HttpService =
```

Comment:
This constructor is there largely because of the core dsl's propensity to generate `PartialFunction`s under common (and I think reasonable) usage. Is there a good reason to remove it?